### DISCUSSION TITLE
Request for recovery of One Time Key

### DISCUSSION THREAD

**Discussion by shahid13a** [twbs/bootstrap#39889] [Q&A] [answered]
### Describe the issue

Dear Sir,
It is requested that I have registered to Omnistock.io. But during my working I have lost my One time key which has been provided by Omnistock.io. 
In view of above it is therefore requested to grant me any source code link so that i could be able to get my lost OneTimeKey.
It is further added that I have been registered to Omnistock.io with mshhahid7211@gmail.com
You are requested to oblidge me at this email if possible please.
regards:
Muhammad Shahid from Pakistan

### Reduced test cases

Hoping for your favorable / kind consideration in the matter please as I have to lost my investment if the same is not considered by your kind side please.

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome, Firefox

### What version of Bootstrap are you using?

Latest Versions

**Comment by julien-deramond** [comment, marked as answer]
Hi @shahid13a 
You're at the wrong place. This space is dedicated to [Bootstrap](https://getbootstrap.com/), not to Omnistock.

**Comment by shahid13a** [comment]
Right Sir,
Can You please let me know about any contact email of OmniStock please

**Comment by JamesVanWaza** [comment]
@shahid13a after looking at their website the email address is [support@omnistock.io](mailto:)

**Comment by shahid13a** [comment]
Sir
Bundle of Thanks I appreciate your co operation

**Comment by shahid13a** [comment]
Dear Sir,
***@***.***
I am not getting any respond from the above email address. It is to request
you that your good office has been approached because you are the
developing company for making their portal and may have any personal
contact information of the OmniStock for Contact.
It is therefore requested that any active email may please be provided if
your good office may have any contact with company please.
